include/PassType.h

```diff
@@ -0,0 +1,51 @@
+#ifndef PASS_H
+#define PASS_H
+
+#include "RunAnalysis.h"
+#include "Utils.h"
+#include "PointerAnalysis.h"
+#include "CFG.h"
+
+#include <set>
+#include <fstream>
+
+typedef std::map<std::string, std::string> AliasMap;
+
+struct MethodsSetHolder {
+    std::set<std::string> methodsSet; 
+    bool setInitialized; 
+};
+
+typedef std::map<std::string, std::map<std::string, MethodsSetHolder>> MappedMethods;
+
+class PassType {
+private: 
+    AliasMap aliasedVars; 
```

Comment:
So `AliasMap` is a mapping between LLVM IR intermediate variables (e.g., %6, %7, %8) and variables visible in the code (e.g., %str). It's used to find all local must-aliases, though it is not expected to find all of them once we begin testing this code on more realistic, larger code 
A comment in the code explaining this will be added